manage.py

```diff
@@ -94,4 +94,8 @@ def prepare_kubernetes_db():
 
 
 if __name__ == "__main__":
+    host_port = app.config['SERVER_NAME'].split(":")
+    host = host_port[0]
+    port = int(host_port[1]) if len(host_port) > 1 else app.config['FALLBACK_PORT']
+    manager.add_command("runserver", Server(use_debugger=True, use_reloader=True, host=host, port=int(port)))
```

Comment:
A minor suggestion, can be changed to 
`int(host_port[1] if len(host_port) > 1 else app.config['FALLBACK_PORT'])`
and no need of `int(port)` will be below